### PR TITLE
Add Pimpkin Special Ability

### PR DESCRIPTION
**Pull request by armandorusso**
Randomly performs its special ability. No VFX added yet, just the main functionality was added.

### CODE REVIEW

**Review by eli-lam — APPROVED** [review]
LGTM!

File: Assets/Scripts/Combat/Abilities/Ability.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (George540) [inline Assets/Scripts/Combat/Abilities/Ability.cs:1]: Remove Unnecessary packages. They throw warnings sometimes

**Review by George540 — APPROVED** [review]
It's good! Just remove useless packages, fields or functions that are not used. They cause warnings

File: Assets/Scripts/Combat/Abilities/Ability.cs (hunk 1 above)
Comment (armandorusso) [reply Assets/Scripts/Combat/Abilities/Ability.cs]: Done! Did it for all other files.

**Pull request merged**